src/main/java/com/aws/greengrass/logmanager/LogManagerService.java

```diff
@@ -230,8 +239,8 @@ private void setUserComponentConfiguration(Map<String, Object> componentConfigMa
                     if (Utils.isNotEmpty(logFileRegexString)) {
                         fileNameRegex.set(Pattern.compile(logFileRegexString));
                     } else {
-                        fileNameRegex.set(Pattern.compile(String.format(DEFAULT_FILE_REGEX,
-                                Pattern.quote(LogManager.getRootLogConfiguration().getFileName()))));
+                        fileNameRegex.set(Pattern.compile(String.format(DEFAULT_FILE_REGEX, Pattern.quote(
+                                Coerce.toString(componentConfigMap.get(COMPONENT_NAME_CONFIG_TOPIC_NAME))))));
                     }
```

Comment:
do you even need this else statement? Further down on line 269 it checks if the regex is set, and does a similar Pattern.compile if not.